web/src/components/storage/ProposalVolumes.jsx

```diff
@@ -214,6 +218,8 @@ const VolumeRow = ({ columns, volume, options, isLoading, onEdit, onDelete }) =>
         <If condition={options.encryption} then={<Em icon={lockIcon}>{_("encrypted")}</Em>} />
         {/* TRANSLATORS: filesystem flag, it allows creating snapshots */}
         <If condition={hasSnapshots} then={<Em icon={snapshotsIcon}>{_("with snapshots")}</Em>} />
+        {/* TRANSLATORS: filesystem flag, system is read only */}
```

Comment:
Serious question. System or filesystem?

Reply:
good question. As it is generic volume, it is filesystem as system can live elsewhere.